server/athenian/api/openapi/openapi.yaml

```diff
@@ -52,11 +52,11 @@ paths:
 # {% endif %}
       x-openapi-router-controller: athenian.api.controllers.user_controller
       x-codegen-request-body-name: body
-  /account/{id}:
+  /account/{id}/members:
     get:
-      operationId: get_account
+      operationId: get_account_members
       parameters:
-      - description: Numeric identifier of the account to list. The user must belong
+      - description: Numeric identifier of the account. The user must belong
```

Comment:
Can we first keep the other endpoint as well so that we do not force webapp to update? This is a breaking change isn't it? This would require bumping api and webapp together from staging to production and the non-atomic rolling update would cause some errors in case of traffic during the deploy.

Reply:
ok, I'll add a redirection :+1:

Reply:
It appears hard to add a redirection for `{id}`-like paths. We will have to deploy together :(

Reply:
Added via duplicating the endpoint in the spec + calling the new handler from the old one in Python.